source/references/glossary.markdown

```diff
@@ -1,82 +1,127 @@
 ---
-title: Glossary
+title: Glossary of Puppet Vocabulary
 layout: default
+canonical: "/references/glossary.html"
```

Comment:
If the canonical link is itself, and it's not a versioned page that'll keep moving forward, then you don't need to add an explicit entry; we automatically set it to self.